front-end/src/SpotifyCallback.jsx

```diff
@@ -65,14 +67,31 @@ function SpotifyCallback() {
                 })
                 .catch(error => {
                     console.error('Error:', error);
+                })
+                .finally(() => {
+                    setTimeout(() => {
+                        setIsLoading(false);
+                    }, 3000);
```

Comment:
any reasoning behind 3000?

Reply:
> any reasoning behind 3000?
Its not too fast or too slow, I tested out different times and decided on 3000.